django_kafka/header.py

```diff
@@ -0,0 +1,9 @@
+from typing import Optional
+
+MessageHeaders = list[tuple[str, str]]
+
+
+def get_header(headers: Optional[MessageHeaders], header: str) -> Optional[str]:
```

Comment:
As discussed in private this gonna be changed to 2 functions or an interface with .list, .get/.first methods as headers may have multiple same “keys”.